### DISCUSSION TITLE
RPA agents for AI chatbot task execution.

### DISCUSSION THREAD

**Discussion by adappterxyz** [akash-network/community#640] [Ideas]
Hey AKT FAM.

 I would like to gather opinions around a more retail facing product such as "RPA agent" as a service. So essentially, its an instance you feed commands to, making it viable to execute web-based tasks easily via API or via chatbots. Agents are essentially containers with a tasks queue that can be preset via 
1) marketplace of playwright scripts
2) api update  

I thought this would be interesting since it can introduce:
a) recurring rental 
b) retail relevance
c) leverage on validated demand ( rabbit r1, it didnt quite work out but i think the execution layer is very much welcomed) 

intro: https://minionprotocol.com/
conceptual vid: https://minionprotocol.com/#demo

Built a very early prototype, but didnt fit into any akashathon track. Would love to hear feedback on such a product.

Happy to hear from y'all $akt fam. 

Kwang / Adappter

**Comment by dominikusbrian** [comment]
I have an application use case of AI-Agents as a services that would be very much love to use the same underlying infrastructure like the RPA Agent as a service being proposed here. Hopefully utilizing the now online LLaMA3 API hosted on Akash, and also the other Akash network infrastructure for scalability. 

Would love to test out the minionprotocol and see how far things already are , so we can understand what stage this proposal is on.  Great demo btw !

Best,
Domi.
